src/components/OpenSeason.js

```diff
@@ -0,0 +1,136 @@
+import { Flex, HStack, Select, Image, Spinner, useToast, Button } from "@chakra-ui/react";
+import { useState } from "react";
+import { db } from '../utils/firebase';
+import { collection, query, getDocs, where, updateDoc } from "firebase/firestore";
+import opensznimg from '../assets/openseason-white.svg';
+
+const OpenSeason = ({ arrayOfAlivePlayers, roomID, handleOpenSzn }) => {
+    const [selectedOpenSeasonPlayer, setSelectedOpenSeasonPlayer] = useState('');
+    const [loading, setLoading] = useState(false);
+    const toast = useToast();
+
+    const handleChange = (event) => {
+        setSelectedOpenSeasonPlayer(event.target.value);
+    };
+
+    const handleClick = async () => {
+        handleOpenSzn(selectedOpenSeasonPlayer);
+        if (!selectedOpenSeasonPlayer) {
+            toast({
+                title: "No player selected.",
+                description: "Please select a player to open season.",
+                status: "warning",
+                duration: 5000,
+                isClosable: true,
+            });
+            return;
+        }
+
+        setLoading(true);
+
+        try {
+            const playerCollectionRef = collection(db, 'rooms', roomID, 'players');
```

Comment:
At some point we should refactor this code to set openSeason to be true and the code below to set openSeason to be false to be one database method which just takes openSeason as a parameter. Don't have to do it now, though